components/RandomFunction.js

```diff
@@ -63,8 +63,8 @@ function RandomFunction({ friendList }) {
       ) : (
         <StyledRandomButton onClick={handleRandomFriendClick}>
           <div>
-            <h1> Be a friend! </h1>
-            <p>Push the button!</p>
+            <h1>Be a friend!</h1>
+            <p>Push the Button</p>
```

Comment:
I'm pretty sure the semantics police go crazy when a headline is nested inside a button.
Please check this on [Caninclue](https://caninclude.glitch.me/)

Apart from that, there are two `<h1>` elements in your DOM tree. Using multiple `<h1>` elements on one page is allowed by the HTML standard (as long as they are not nested), but this is not considered a best practice. 🤓

Reply:
Understood